database/migrations/2023_11_12_183753_meeting_data.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+
+    private string $meetingCategoryDictionaryId = 'ce12aa23-a5db-49f3-987b-d2db3ab24a3b';
+    private string $meetingTypeDictionaryId = '4cc5d6b5-5913-4e07-9fcb-09b6f3d3d2a0';
+    private string $meetingStatusDictionaryId = '3865a3c3-0038-4668-9d55-5d05b79d7fcd';
+    private string $meetingResourceDictionaryId = 'fdb0f818-aa1e-4ed9-97cc-2a3cb1b702df';
+    private string $meeting_category_attribute_id = '8111626d-130c-454d-b0c0-9fda9ab9917a';
+    private string $meeting_type_attribute_id = '5f7d5e66-03f9-4bcd-a726-fde82cf98d6f';
+    private string $meeting_status_attribute_id = 'e443e2c2-82fc-41d3-8fda-fe374e5329d3';
+    private string $meeting_resource_attribute_id = 'ff2fb31e-c992-4e2d-b543-f2768ee1c897';
+
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('meetings', function (Blueprint $table) {
+            $table->char('id', 36)->collation('ascii_bin')->primary();
+            $table->char('facility_id', 36)->collation('ascii_bin');
+            $table->char('category_dict_id', 36)->collation('ascii_bin');
+            $table->char('type_dict_id', 36)->collation('ascii_bin');
+            $table->string('name')->nullable();
+            $table->text('notes')->nullable();
+            $table->date('date');
+            $table->integer('start_minutes');
+            $table->integer('minutes');
```

Comment:
duration_minutes może?

Reply:
"spotkanie ma 60 minut", czy "spotkanie trwa 60 minut" dość na to samo wychodzi, a tak jest krócej

Reply:
Tylko że "a.b" oznacza coś jakby "b of a", a "duration of meeting" jest zdecydowanie bardziej poprawne niż "minutes of meeting". A za tym "duration" jest suffix "_minutes" po prostu jako podanie jednostki, dlatego "duration_minutes". Podobnie jak mielibyśmy raczej "client.age" niż "client.years", mimo że zdecydowanie można powiedzieć "mieć X lat" i to jest jednoznaczne.

Bardzo się nie upieram jednakowoż.

Reply:
Poszedłem w duration_minutes - trochę lepiej tłumaczy wartość w kolumnie.